package.json

```diff
@@ -10,23 +10,24 @@
     "lint": "eslint --cache --cache-location ./node_modules/.cache/eslint .",
     "format": "prettier --write .",
     "validate": "run-p \"test -- run\" lint typecheck test:e2e:run",
-    "knex": "knex",
+    "kysely": "kysely",
     "": "",
-    "start:migrate": "knex migrate:latest",
+    "start:migrate": "kysely migrate:latest",
     "prepare": "husky install",
     "typecheck": "tsc -b && tsc -b cypress",
-    "build:css": "npm run generate:css --minify",
+    "build:css": "run-s generate:css --minify",
```

Comment:
```suggestion
    "build:css": "run-s generate:css -- --minify",
```

Reply:
i didn't mean to commit some of these, ill send a follow-up with some of these fixes